src/golang/lib/repos/tests/seed.go

```diff
@@ -12,6 +12,27 @@ const (
 	testOrgID = "aqueduct-test"
 )
 
+// seedNotification creates count notification records.
+func (ts *TestSuite) seedNotification(count int) []models.Notification {
+	notifications := make([]models.Notification, 0, count)
+
+	for i := 0; i < count; i++ {
+		receiverID := uuid.New()
```

Comment:
I dont think sqlite will throw an error but technically the receiverID is a foreign key with the users table, so lets either require a user to be passed into this seed fn or generate a random user to use. Can you comment which approach you take as part of the function comment here